components/application_manager/application_manager.h

```diff
@@ -344,7 +343,7 @@ class ApplicationManager {
      * @param app The app to run.
      * @return True on success, false on failure.
      */
-    bool RunApp(const App &app);
+    void RunApp(App *app);
```

Comment:
RunApp() is a helper method. It exists to avoid duplicate code so it should never be null. The App's copy constructor and copy assignment operator are deleted in order to prevent confusion with the App ids.

Reply:
Resolved in meeting